README.md

```diff
@@ -103,6 +103,15 @@ This is a common issue with latest PHP versions under windows (related to a inco
 - download this CAs database `https://curl.haxx.se/ca/cacert.pem` to `c:/cacert.pem`
 - you need to edit your php.ini and add `openssl.capath=c:/cacert.pem` (it should point to the file you downloaded)
 
+> I am not using composer, my host does not allow using Composer
+
+This package uses composer for mantianing dependencies. However, if you cannot use composer for some reasons. Please follow the following steps and upload these dependencies manually.
```

Comment:
`However, if you cannot use composer for some reasons` does not make sense at all. This is only when the user can not run composer on the server.